### PR TITLE
Add aneurysm test

### PR DESCRIPTION
**Pull request by keiyamamo**
add test for aneurysm problem file

### CODE REVIEW

File: src/fsipy/simulations/aneurysm.py [hunk 1]
```diff
@@ -64,15 +67,16 @@ def set_problem_parameters(default_variables, **namespace):
...
+        fsi_region=[0.123, 0.134, 0.063, 0.004],  # x, y, and z coordinate of FSI region center,
+                                                  # and radius of FSI region sphere
         # Simulation parameters
         folder="aneurysm_results",  # Folder name generated for the simulation
         mesh_path="mesh/file_aneurysm.h5",
         FC_file="FC_MCA_10",  # File name containing the fourier coefficients for the flow waveform
         P_FC_File="FC_Pressure",  # File name containing the fourier coefficients for the pressure waveform
         compiler_parameters=_compiler_parameters,  # Update the defaul values of the compiler arguments (FEniCS)
         save_deg=2,  # Degree of the functions saved for visualisation
+        scale_probe=True,  # Scale the probe points to meters
```
Comment (johannesring) [inline src/fsipy/simulations/aneurysm.py:79]: Why is this needed?

File: tests/test_simulations.py [hunk 2]
```diff
@@ -129,3 +130,69 @@ def test_cylinder_problem(input_mesh, tmpdir):
...
+    assert output_match_flow_rate is not None, "Regular expression did not match the output."
+
+    expected_flow_rate = 7.297633240079062e-10
+    flow_rate_at_inlet = float(output_match_flow_rate[-1])
+
+    print(f"Flow rate: {flow_rate_at_inlet}")
+    assert np.isclose(flow_rate_at_inlet, expected_flow_rate), "Flow rate does not match expected value."
+
+    # check velocity mean, min, max in the domain
+    ourput_velocity = (r"Velocity \(mean, min, max\): (\d+(?:\.\d+)?(?:e-\d+)?)\s*,\s*(\d+(?:\.\d+)?(?:e-\d+)?)\s*,"
```
Comment (johannesring) [inline tests/test_simulations.py:156]: Should it be `output_velocity`?

File: tests/test_simulations.py [hunk 3]
```diff
@@ -129,3 +130,69 @@ def test_cylinder_problem(input_mesh, tmpdir):
...
+                       r"\s*(\d+(?:\.\d+)?(?:e-\d+)?)")
+
+    output_match_velocity = re.findall(ourput_velocity, str(result))
```
Comment (johannesring) [inline tests/test_simulations.py:159]: Same here.

**Review by johannesring — APPROVED** [review]
Looks fine.

File: src/fsipy/simulations/aneurysm.py (hunk 1 above)
Comment (johannesring) [reply src/fsipy/simulations/aneurysm.py:79]: I see now.

**Comment by keiyamamo** [comment]
@johannesring  Thank you for the review!
